src/metrics/definitions.md

```diff
@@ -0,0 +1,9 @@
+# Metric Definitions
```

Comment:
I wonder if this would better go in the terminology document on docs.tmo? We can always split that up into different types of terminology, adding a metrics section.

https://docs.telemetry.mozilla.org/concepts/terminology.html

Maybe better yet, we could add a section per term, so we could do something like like this:

https://docs.telemetry.mozilla.org/concepts/terminology.html#usage-criterion
https://docs.telemetry.mozilla.org/concepts/terminology.html#dimension

Then, when talking about / introducing these terms in the metrics section, we can link back to these terms.

Reply:
yup, good point!  will do :)

Reply:
Looking at this now, I think it might be useful to explain what a metric, usage criteria, and slicing dimension is "in situ".  It may not be obvious to people how to use this section.  Perhaps we can have short notes here and also more thorough definitions in the terminology doc?

Reply:
I think that could work, as long as the definitions in-situ are kept short. I assume at least the short definitions are unlikely to change.

Reply:
Hmm, I guess I don't really see the point of a document that's just an abbreviated version of another? 

I think it would make more sense to provide some introductory content here on what you're trying to accomplish with the different types of detailed definitions in the child documents, then link to them explicitly here (as sort of a meta table of contents, [like the bigquery cookbook now does](https://docs.telemetry.mozilla.org/cookbooks/bigquery.html)). IMO some kind of story needs to be told about why people should read this section closely and carefully -- it doesn't need to be long (in fact it's probably better if it isn't).

Reply:
That's a fair point.  I'll make a pass of making this page more useful and am open to eventually removing it if we prefer to just promote its children one level.